packages/glibc.rb

```diff
@@ -3,20 +3,24 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version LIBC_VERSION
   license 'LGPL-2.1+, BSD, HPND, ISC, inner-net, rc, and PCRE'
-  compatibility 'all'
 
   depends_on 'gawk' => :build
   depends_on 'libgd' => :build
   depends_on 'libidn2' => :build
   depends_on 'texinfo' => :build
+  depends_on 'hashpipe' => :build
 
   # enable-obsolete-rpc should NOT be included as a build option as
   # this functionality is provided by libtirpc
 
-  case LIBC_VERSION
+  @LIBC_VERSION = LIBC_VERSION
```

Comment:
Shouldn't need to do this since `LIBC_VERSION` is a constant.

Reply:
Yes, but the plumbing for this is just in there for bootstrapping builds of newer glibc on older glibc. Otherwise it's a huge pain.

Reply:
As suggested, only using LIBC_VERSION.